dkpro-core-opennlp-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/opennlp/OpenNlpChunker.java

```diff
@@ -55,6 +58,7 @@
 /**
  * Chunk annotator using OpenNLP.
  */
+@MimeTypeCapability(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)
```

Comment:
I would reserve the MimeTypeCapability to input files and output files. The ResourceParameter annotation below on the modelLocation parameter should be sufficient.